app/models.py

```diff
@@ -1168,7 +1168,7 @@ class Brief(db.Model):
     clarification_questions = db.relationship(
         "BriefClarificationQuestion",
         order_by="BriefClarificationQuestion.published_at",
-        lazy='joined'
+        lazy='select',
```

Comment:
Why are we switching to `select` here? We're still loading them for the list briefs endpoint, won't this result in N+1 queries being sent?